### PR TITLE
Solved Til The Modules Section

### PR DESCRIPTION
**Pull request by Moyo75**
I did til the modules section and noticed that there was nothing much after that. And I also corrected spelling errors in all readme.md files.

### CODE REVIEW

File: project-javascript-i1-iterators/README.md [hunk 1]
```diff
@@ -41,16 +41,16 @@ If you are not familier with them please download Learn JavaScript eBook from ht
...
-7. Filter items that start with `c` charachter and them as ordered list into console with composing .filter(), .map() and .forEach().
+7. Filter items that start with `c` charachter and then as ordered list log them into console with composing .filter(), .map() and .forEach().
 
-8. Make array `actionsLetters` that holds number of letters in each item of the array(Note: do not count spaces).
+8. Make array `actionsLetters` that holds number of letters in each item of the array (Note: do not count spaces).
 
-9. Count sum of letters of all items in the array. Use .reduce() and the array of previous step.
+9. Do the sum of letters of all items in the array. Use .reduce() on the array of previous step.
 
-10. Find first item of `actionsLetters` that has less than 10 letters. Use .filterIndex
```
Comment (basir) [inline project-javascript-i1-iterators/README.md]: Thanks

File: project-javascript-j1-objects/JohnAdemoye.html [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+        universtiy4: { gpa: 3.0, year: 2005 }
+      };
+
+      // 10
+      for (university in employee.educations) {
+        console.log(university);
+      }
+
+      // 11
+      const gpas = Object.keys(employee.educations).map(
```
Comment (basir) [inline project-javascript-j1-objects/JohnAdemoye.html]: Perfect

File: project-javascript-k1-advanced-objects/JohnAdemoye.html [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+            `Name: ${this.firstName} ${this.lastName} ID:${this.id} Major: ${this.major} GPA: ${this.gpa}`
+          );
+        },
+
+        // 4
+        get fullName() {
+          if (!this.firstName) {
+            return `Mr/Mrs ${this.lastName}`;
+          }
+          return `Mr/Mrs ${this.firstName} ${this.lastName}`;
```
Comment (basir) [inline project-javascript-k1-advanced-objects/JohnAdemoye.html]: Based on question:
"If first name is empty return Mr/Ms {last name} else return {firstName} {lastName}."
So the else part:
 return ` ${this.firstName} ${this.lastName}`;

File: project-javascript-l1-classes/JohnAdemoye.html [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+          return false;
+        }
+      }
+
+      // 5
+      const myTL = new TrafficLight('red', 'Avenue 1');
+
+      // 8/9
+      class VIPTrafficLight extends TrafficLight {
+        constructor(SpecialEffects) {
```
Comment (basir) [inline project-javascript-l1-classes/JohnAdemoye.html]:
```
constructor(color, address, specialEffects){
                super(color, address);
                this.specialEffects = specialEffects;
                }
```

File: project-javascript-k1-advanced-objects/JohnAdemoye.html (hunk 3 above)
Comment (Moyo75) [reply project-javascript-k1-advanced-objects/JohnAdemoye.html]: Oh! OK. Thanks.

File: project-javascript-l1-classes/JohnAdemoye.html (hunk 4 above)
Comment (Moyo75) [reply project-javascript-l1-classes/JohnAdemoye.html]: OK. Thanks.

File: project-javascript-d1-functions/JohnAdemoye.html [hunk 5]
```diff
@@ -46,20 +49,24 @@
...
- 
+
+      // 8
       function nextYear(year) {
         return year + 1;
       }
 
       console.log(formatDate(nextYear(year), month, day));
 
+      // 9
```
[the remark this answers is not in the document]
Comment (Moyo75) [reply project-javascript-d1-functions/JohnAdemoye.html]: OK. I'll do that.

File: project-javascript-d1-functions/JohnAdemoye.html [hunk 6]
```diff
@@ -46,20 +49,24 @@
         return dayNumber;
       }
 
+      // 6
       function formatDate(year, month, day) {
```
[the remark this answers is not in the document]
Comment (Moyo75) [reply project-javascript-d1-functions/JohnAdemoye.html]: OK. I'll do that.

File: project-javascript-g1-loops/JohnAdemoye.html [hunk 7]
```diff
@@ -4,37 +4,43 @@
...
+      // 7
       const fruits = [
         ['apple', 'orange', 'banana'],
         ['coconut', 'cumbersome'],
         ['starbery', 'bluebery']
       ];
 
+      // 8
       for (let index = 0; index < fruits.length; index++) {
         for (let index2 = 0; index2 < fruits[index].length; index2++) {
```
[the remark this answers is not in the document]
Comment (Moyo75) [reply project-javascript-g1-loops/JohnAdemoye.html]: Thank you

File: project-javascript-h1-hof/JohnAdemoye.html [hunk 8]
```diff
@@ -0,0 +1,33 @@
...
+      const repeatDuration = (repeat, times, func, param) => {
+        const start = new Date();
+        repeat(func, param, times);
+        const end = new Date();
+        console.log(`Running time is ${end - start}ms`);
+      };
+
+      // 5
+      const factorial = num => {
+        num > 1 ? num * factorial(num - 1) : 1;
```
[the remark this answers is not in the document]
Comment (Moyo75) [reply project-javascript-h1-hof/JohnAdemoye.html]: Thanks!

File: project-javascript-e1-scope/JohnAdemoye.html [hunk 9]
```diff
@@ -23,9 +30,11 @@
...
 
       logColor1();
-      logColor2('red');
+
+      // 9
       logColor3('green');
 
+      // 10
       // ANALYSIS
       // The color variable has a global scope, thus it can be accessed by the different logColor functions. However, the color variable declared and assigned "light green" in the logColor3 function has the local scope of it's function. Therefore, it cannot be accessed by the other logger functions.
```
[the remark this answers is not in the document]
Comment (Moyo75) [reply project-javascript-e1-scope/JohnAdemoye.html]: Thanks

File: project-javascript-j1-objects/JohnAdemoye.html (hunk 2 above)
Comment (Moyo75) [reply project-javascript-j1-objects/JohnAdemoye.html]: Thanks

**Pull request merged**